modules/completion/Makefile

```diff
@@ -0,0 +1,13 @@
+BASH_COMPLETION_FLAG ?= ~/.make.bash.completion
+
+completion:
```

Comment:
I think this should be in `make` module, so `make make/completion`

Reply:
`completion/bash` is better for me
What do you think?